testutil-server/src/test/proto/spine/testing/server/log/events.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package spine.testing.server.log;
+
+import "spine/options.proto";
+
+option (type_url_prefix) = "type.spine.io";
+option java_package = "io.spine.testing.server.log";
+option java_outer_classname = "EventsProto";
+option java_multiple_files = true;
+
+message FloatLogged {
```

Comment:
It's weird to have `LogInteger` command with no related event and `FloatLogged` event with no origin command.